src/components/views/MyMilestones.jsx

```diff
@@ -441,20 +396,50 @@ class MyMilestones extends Component {
                   </p>,
                 );
               })
-              .catch(err => {
-                if (txHash && err.message && err.message.includes('unknown transaction')) return; // bug in web3 seems to constantly fail due to this error, but the tx is correct
-                ErrorPopup(
-                  'Something went wrong with the transaction. Is your wallet unlocked?',
-                  `${etherScanUrl}tx/${txHash}`,
-                );
+              .catch(e => {
+                ErrorPopup('Something went wrong with cancelling your milestone', e);
```

Comment:
This is a transaction, it should still have the txhash passed to error component as it did before.

Reply:
I'll just revert the changes as per https://github.com/Giveth/giveth-dapp/pull/443/files#r206610484 @vojtechsimetka?

Reply:
Fixed this. :+1: